src/components/Sponsors.tsx

```diff
@@ -1,17 +1,27 @@
 import React, { CSSProperties } from "react";
 import { Grid, Image, Header, Segment } from "semantic-ui-react";
 
-export function Sponsors() {
+type Props = {
+  show?:boolean
+}
+
+export function Sponsors(props: Props) {
   const image__css: CSSProperties = {
     margin: "0 auto",
     maxWidth: "250px",
     padding: 0,
   };
+  const showHeader = onAbout();
 
+  function onAbout() {
+    if(props.show == null) {
```

Comment:
Although this code is technically a _correct_ way to use `==` operator for the intended of checking the type is `null`, I would prefer we use `===` everywhere due to the confusion that can arise in all of these different cases:

```js
const aNull = null

if (aNull == null) {
	console.log('always true')
}

=>  always true

if (aNull === null) {
	console.log('always true')
}
=>  always true

if (aNull) {
	console.log('always false')
}

=> undefined
```

Reply:
Fixed this